motor/core.py

```diff
@@ -1670,6 +1773,37 @@ class AgnosticCommandCursor(AgnosticBaseCursor):
 
     _CommandCursor__die = AsyncRead()
 
+    def _lazy_init(self):
+        if not self.delegate:
+            self.delegate = self._target.delegate.watch(**unwrap_kwargs_session(self._kwargs))
```

Comment:
This is not needed for AgnosticCommandCursor.

Reply:
Removed